cmd/astrolabe/main.go

```diff
@@ -0,0 +1,312 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"github.com/pkg/errors"
+	"github.com/urfave/cli/v2"
+	"github.com/vmware-tanzu/astrolabe/gen/client"
+	"github.com/vmware-tanzu/astrolabe/pkg/astrolabe"
+	client2 "github.com/vmware-tanzu/astrolabe/pkg/client"
```

Comment:
can we refactor the two clients to be, like "genClient" and "pkgClient"?

client and client2 are really confusing when reading the code.

Reply:
OK